subsys/random/rand32_ctr_drbg.c

```diff
@@ -51,6 +55,15 @@ static int ctr_drbg_initialize(void)
 {
 	int ret;
 
+	entropy_dev = DEVICE_DT_GET(DT_CHOSEN(zephyr_entropy));
+	if (!entropy_dev) {
```

Comment:
This condition can never hold true, DEVICE_DT_GET always returns a non-null reference (otherwise it would result in a build failure).

Reply:
Fair enough. Actually better if there is a build failure. So I'll take out the !entropy_dev, which was similar to what it was before.